### PR TITLE
feat(quantic|atomic): update genQA answer section title

### PR DESCRIPTION
**Pull request by erocheleau**
SFINT-5425 | SVCC-3509

Changing from 

"Generated answer for you" -> "Generated answer"

I guess it should be updated also for all the translations in atomic?

### CODE REVIEW

**Comment by olamothe** [comment]
> [SFINT-5425](https://coveord.atlassian.net/browse/SFINT-5425) | [SVCC-3509](https://coveord.atlassian.net/browse/SVCC-3509)
> 
> Changing from
> 
> "Generated answer for you" -> "Generated answer"
> 
> I guess it should be updated also for all the translations in atomic?

Yeah, since it's only one value you can do it "manually" on google translate, or here: https://demos.coveodemo.com/gpt/index.html

File: packages/atomic/src/locales.json [hunk 1]
```diff
@@ -6536,7 +6536,7 @@
     "zh-TW": "產生的答案被隱藏"
   },
   "generated-answer-title": {
```
Comment (olamothe) [inline packages/atomic/src/locales.json:6538]:
```json
  "generated-answer-title": {
    "en": "Generated answer",
    "fr": "Réponse générée",
    "cs": "Vygenerovaná odpověď",
    "da": "Genereret svar",
    "de": "Erzeugte Antwort",
    "el": "Δημιουργημένη απάντηση",
    "es": "Respuesta generada",
    "fi": "Luotu vastaus",
    "hu": "Generált válasz",
    "id": "Jawaban yang dihasilkan",
    "it": "Risposta generata",
    "ja": "生成された回答",
    "ko": "생성된 답변",
    "nl": "Gegenereerd antwoord",
    "no": "Generert svar",
    "pl": "Wygenerowana odpowiedź",
    "pt": "Resposta gerada",
    "pt-BR": "Resposta gerada",
    "ru": "Сгенерированный ответ",
    "sv": "Genererat svar",
    "th": "คำตอบที่สร้างขึ้น",
    "tr": "Üretilen yanıt",
    "zh": "生成的答案",
    "zh-CN": "生成的答案",
    "zh-TW": "產生的答案"
  }
```

**Review by olamothe — APPROVED** [review]

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml
Comment (mmitiche) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:1]: We also need to update the french translation of this Quantic label here: https://github.com/coveo/ui-kit/blob/ac15f5c11c89fd6af30a92d6abf7c6a5b984b8e5/packages/quantic/force-app/main/translations/fr.translation-meta.xml#L676

Comment (jelmedini) [reply packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:1]: @MehdiMitiche  I will do it for Etienne as he is not here, i ve already updated atomic part of his branch

Comment (mmitiche) [reply packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:1]: Perfect thank you!

**Pull request merged**